### PR TITLE
[POA-1089] Add check for bridge networking in 'ecs add' command

### PR DESCRIPTION
**Pull request by mudit-postman**
Plan: https://postmanlabs.atlassian.net/wiki/spaces/PO/pages/5053218962/Plan+Insights+Agent+ECS+and+other+tasks+for+supporting+50+users#Alert-user-about-the-wrong-network-mode-in-ecs-add.1

Added bridge networking mode check in the `getTaskState()` function in the `ecs add` command and exit installation with proper error message 

---
Reviewers:
* Primary: @shreys7 
* Secondary: @mgritter

### CODE REVIEW

**Review by shreys7 — APPROVED** [review]

File: cmd/internal/ecs/add.go [hunk 1]
```diff
@@ -520,6 +520,14 @@ func getTaskState(wf *AddWorkflow) (nextState optionals.Optional[AddWorkflowStat
 		return awf_error(errors.Errorf("Error while loading ECS task definition; please contact %s for assistance.", consts.SupportEmail))
 	}
 
+	// Check for bridge networking mode.
+	if output.NetworkMode == types.NetworkModeBridge {
+		printer.Errorf("This task definition is using bridge mode for networking which requires running Insights Agent as a daemon service. " +
+			"However, this is not currently supported by \"ecs add\" command. Please refer to documentation for running Insights Agent as a daemon service, " +
+			"https://learning.postman.com/docs/insights/insights-gs/#configure-the-insights-agent-as-a-daemon-service")
```
Comment (shreys7) [inline cmd/internal/ecs/add.go:527]: For some reason, learning center scrolls down to this section when I open the link you added here 
image

Comment (mudit-postman) [reply cmd/internal/ecs/add.go:527]: Yes, I will report this to Nermina with a video today.

Comment (mudit-postman) [reply cmd/internal/ecs/add.go:527]: This is likely due to a delay in image load. The browser goes to the point instantly, and the actual point gets shifted when images load.
If you try to reload it repeatedly, the images are fetched from the cache and hence, the page loads correctly.

File: cmd/internal/ecs/add.go [hunk 2]
```diff
@@ -520,6 +520,14 @@ func getTaskState(wf *AddWorkflow) (nextState optionals.Optional[AddWorkflowStat
 		return awf_error(errors.Errorf("Error while loading ECS task definition; please contact %s for assistance.", consts.SupportEmail))
 	}
 
+	// Check for bridge networking mode.
+	if output.NetworkMode == types.NetworkModeBridge {
+		printer.Errorf("This task definition is using bridge mode for networking which requires running Insights Agent as a daemon service. " +
```
Comment (mgritter) [inline cmd/internal/ecs/add.go:525]:
```suggestion
		printer.Errorf("This task definition is using bridge mode for networking, which requires running Insights Agent as a daemon service. " +
```

**Review by mgritter — APPROVED** [review]

**Pull request merged**